queries/swift/locals.scm

```diff
@@ -19,3 +64,40 @@
   (class_declaration)
   (protocol_declaration)
 ] @local.scope
+
+; References
+(statements
+  (assignment
+    (directly_assignable_expression
+      (simple_identifier) @local.reference)))
+
+(navigation_expression
+  target: (self_expression)
+  suffix:
+    (navigation_suffix
+      suffix: (simple_identifier) @local.reference))
+
+(navigation_expression
+  target: (simple_identifier) @local.reference)
+
+(call_expression
+  (simple_identifier) @local.reference)
+
+(prefix_expression
+  (simple_identifier) @local.reference)
+
+(lambda_literal
+  (capture_list
+    (capture_list_item
+      name: (simple_identifier) @local.reference)))
+
+(switch_statement
+  expr: (simple_identifier) @local.reference)
+
+(self_expression) @local.reference
+
+condition: (simple_identifier) @local.reference
+
+collection: (simple_identifier) @local.reference
+
+value: (simple_identifier) @local.reference
```

Comment:
I noticed that some queries declare just:
```
(simple_identifier) @local.reference
```

However, I would like to avoid capturing things that are declarations, so I guess this is a better approach?